Makefile

```diff
@@ -48,7 +48,7 @@ test: ## Run tests with all features and without default features.
 	cargo test --all-targets --no-default-features  --no-fail-fast --release
 
 check-release: ## Check that the release build compiles.
-	cargo release --workspace --no-push --no-tag \
```

Comment:
why was `--no-publish` not needed before?

Reply:
Not sure tbh. But now it throws an error as the first package (`ibc-primitives`) is not published yet.